reviewrot-cronjob-template.yaml

```diff
@@ -0,0 +1,102 @@
+kind: Template
+apiVersion: v1
+metadata:
+  name: code-review-reminder-cronjob-template
+  annotations:
+    openshift.io/display-name: "Code review reminder cronjob"
+    description: "Description"
+    tags: "reviewrot code-review-reminder cronjob"
+message: |
+  Cronjob for Code review reminder will run by a schedule ${SCHEDULE}
+  with secrets ${SECRET_CONFIG}/${CONFIG} and
+  ${SECRET_CERTIFICATE}/${CERTIFICATE} on the image
+  quay.io/pbortlov/reviewrot:latest.
+parameters:
+- description: Owner login
+  name: OWNER
+  required: true
+- description: CPU limit
+  name: CPU_LIMIT
+  required: true
+  value: 200m
+- description: Memory limit
+  name: MEMORY_LIMIT
+  required: true
+  value: 256Mi
+- description: Name of configuration's secret
```

Comment:
Why is configuration a secret and not a config map?

Reply:
There can be API tokens inside a config file.